tcmur_cmd_handler.c

```diff
@@ -1647,24 +1698,29 @@ caw_init_readcmd(struct tcmulib_cmd *origcmd, size_t length)
 	state->requested = length;
 	state->origcmd = origcmd;
 
-	readcmd->cmdstate = state;
+	tcmu_cmd_set_private(readcmd, read_rcmd_state);
+	read_rcmd_state->cmdstate = state;
```

Comment:
Maybe this line should move ahead of tcmu_cmd_set_private().

Reply:
OK, i just followed the coding style in this file. :) Will fix this and the similar below.